config/unicorn.rb

```diff
@@ -0,0 +1,12 @@
+worker_processes 4
+stdout_path "#{ENV.fetch('GOVUK_APP_LOGROOT')}/app.out.log"
+stderr_path "#{ENV.fetch('GOVUK_APP_LOGROOT')}/app.err.log"
+before_exec do |server|
+    ENV["BUNDLE_GEMFILE"] = "#{ENV.fetch('GOVUK_APP_ROOT')}/Gemfile"
```

Comment:
This is a straight copy of [the file being loaded here](https://github.gds/gds/alphagov-deployment/blob/master/signonotron2/to_upload/unicorn.rb#L4)
Admittedly I didn't employ too much thinking about the contents.